controller_manager:
  ros__parameters:
    update_rate: 30

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    terrence_controller:
      type: "terrence_controller/TerrenceController"

terrence_controller:
  ros__parameters:
    left_joint_name: "DS_Joint"
    right_joint_name: "PS_Joint"

    wheel_radius_m: 0.085
    wheel_separation_m: 0.42

    max_wheel_radps: 10.0
    cmd_timeout_s: 0.25
    moving_eps_radps: 0.05

    odom_topic: "/odom"
    odom_frame_id: "odom"
    base_frame_id: "base_link"
    publish_odom_tf: true
